both/access_control.js

```diff
@@ -258,38 +257,24 @@ if(Meteor.isServer) {
         'updatedAt',
         'createdAt',
       ];
-      return _onlyAllowedFields(fields, allowedFields);
+      return onlyAllowedFields(fields, allowedFields);
     },
     fetch: [ 'competitionId' ],
   });
 
   Registrations.allow({
     insert: function(userId, registration) {
-      return Registrations.userIsAllowed(userId, registration);
+      return Registrations.allowOperation(userId, registration, []);
```

Comment:
Ah! Thanks for checking this. I believe **checkedInEvents** is a red herring, as we specify a default value for it in the SimpleSchema.

**countryId**, **gender**, and **dob** do seem like real problems, however. I guess the idea here is that people set these fields when they register, but they can't change them afterwards? I guess I kind of like that behavior (this way someone can't change their country after the organizers have sanity checked all the registrations, for instance).

Agreed that this is out of the scope of this PR. Lets leave the registration code unstrict. I've created #216 to track patching this hole.